src/swarmus_ros_navigation/launch/navigation.launch

```diff
@@ -7,7 +7,7 @@
   <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
     <!-- Load common parameters of the local and global planner-->
     <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/>
-    <param name="base_global_planner" value="global_planner/GlobalPlanner"/>
+    <!--param name="base_global_planner" value="global_planner/GlobalPlanner"/-->
```

Comment:
We completely remove global planning?

Reply:
@etienn8

Reply:
The global planner is not completly removed. In this case, the legacy NAVfn planner is used since its the default one. In practice we had problems with the GlobalPlanner version so we returned back to the old planner which had better perfromances in our demos